styles/alert.css

```diff
@@ -22,3 +22,102 @@
   --bs-alert-color: rgb(var(--stanford-illuminating-dark-rgb));
   --bs-alert-border-color: rgb(var(--stanford-illuminating-dark-rgb));
 }
+
+/* Banner styles */
+.banner {
+  --bs-banner-bg: none;
+  --bs-banner-padding-x: 16px;
+  --bs-banner-padding-y: 16px;
+  --bs-banner-margin-bottom: 1rem;
+  --bs-banner-color: inherit;
+  --bs-banner-border-color: transparent;
+  --bs-banner-border: var(--bs-border-width) solid var(--bs-banner-border-color);
+  --bs-banner-border-radius: none;
+  --bs-banner-link-color: inherit;
```

Comment:
I don't see these variables defined in bootstrap, so we shouldn't us the `bs-` namespace.

Reply:
@jcoyne would you recommend I maintain them as variables with a different namespace like `--stanford-banner-*` or just use the values in place of the variables were used?

Reply:
I'd just use `--banner-border-color` or similar.